moped-editor/src/components/Maps/ComponentsDrawControl.js

```diff
@@ -21,7 +18,6 @@ export const DrawControl = React.forwardRef((props, ref) => {
         props.overrideDirectSelect();
       });
       map.on("draw.selectionchange", props.onSelectionChange);
-      map.on("load", props.initializeExistingDrawFeatures);
```

Comment:
Turns out this wasn't the right way to add existing features into the draw tools. This is handled with `useExistingDrawnFeatures` now.